pkg/certificateauthority/handler.go

```diff
@@ -42,7 +42,7 @@ func (h *configmapHandler) Handle(ctx context.Context, in *corev1.ConfigMap) err
 	caBundle := in.Data[caBundleKey]
 
 	// Retrieve the Certificate Authority bundle from Disk.
-	// If an error is returned and is not related to a nonexistant file, return an error.
+	// If an error is returned and is not related to a non-existent file, return an error.
```

Comment:
nit: we shouldn't include this change in this pr since it seems unrelated.

Reply:
Addressed.